app/controllers/api/v1/keywords_controller.rb

```diff
@@ -22,6 +22,18 @@ def create
         end
       end
 
+      def show
+        keyword = policy_scope(Keyword).find(params[:id])
```

Comment:
Can we just use?

```suggestion
        keyword = Keyword.find(params[:id])
```

Reply:
The idea is to show only keyword that the user owned, hence the pundit scope. 😄

Reply:
ah right, sorry i miss it.